roles/change-paths/tasks/main.yml

```diff
@@ -214,6 +163,19 @@
     - "{{ link_attributes }}"
   notify: Clear site cache
 
+# We already replaced relative references in href and src attributes (like
+# src="/methane/), but we only did it if the vhost is defined, because of the
+# edge case where inventory_hostname also is a legitimate path for an href.
+# Here we more broadly replace the "/inventory_hostname/sites/default/files" string
```

Comment:
So the example here would be: https://sites.stanford.edu/books/books/sites/default/files/book.jpg?

Reply:
Yes, if that's the site. But the actual code replacement would be:
Before: `src="/books/sites/default/files/book.jpg`
After: `src="/sites/g/files/sbiybj5000986/f/book.jpg`